Tests/MapKitTests/MKPolylineTests.swift

```diff
@@ -24,13 +24,27 @@ final class MKPolylineTests: XCTestCase {
         (37.329767, -121.885813)
         ].map(CLLocationCoordinate2D.init)
 
-    func testInitWithCoordinates() {
-        var refCoordinates = coordinates
+    let locations = [
```

Comment:
You're just copying the coordinates above, which isn't necessary. I think what you should do is remove `.map(CLLocationCoordinate2D.init)` for `coordinates` above, then use that instead in `testInitWithCoordinates`. Similarly, you would then use `coordinates.map(CLLocation.init)` in `testInitWithLocations`. Does that make sense?